source/module_esolver/esolver_ks.cpp

```diff
@@ -4,6 +4,8 @@
 #include "../src_io/print_info.h"
 #ifdef __MPI
 #include "mpi.h"
+#else
+#include "chrono"
```

Comment:
why do you add this header here?
There is no addition code in this cpp file without this header.

Reply:
The serial code line 185, 243 need this header.